### ISSUE TITLE
female undead model getting stucked everywhere

### ISSUE THREAD

**Issue by tehelelol** [Atlantiss/NetherwingBugtracker#5025]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
female undead model getting stucked everywhere

**Current behaviour**:
female undead model getting stucked everywhere
for example: 
tomb in ruins of lordereon
every small stair (example battlemasters in shattrath, you cant run up the stairs without getting stuck completly.)
 basically every small height increase makes you stuck.

**Expected behaviour**: 
should be able to run up stairs smoothly
shouldn't get stuck different then other races/models

**Server Revision**:

**Comment by floridalady** [comment]
ur not only one..stairs n stuff here fucks me up too..  u need to jump to solve it

**Comment by Evolvee** [comment]
clientside
